### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/how-to/graphql.md`

### PR DESCRIPTION
**Pull request by AnandaCampelo**
🌐 Add Portuguese translation for `docs/pt/docs/how-to/graphql.md`

### CODE REVIEW

**Review by JoaoGustavoRogel — CHANGES_REQUESTED** [review]
Great text!!

File: docs/pt/docs/how-to/graphql.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# GraphQL
+
+Como **FastAPI** é baseado no padrão **ASGI**, é muito fácil integrar qualquer biblioteca **GraphQL** também compatível com ASGI.
+
+Você pode combinar *operações de caminho* normais do FastAPI com GraphQL na mesma aplicação.
+
+/// tip
```
Comment (JoaoGustavoRogel) [inline docs/pt/docs/how-to/graphql.md:7]: You can add translation for this:
```
/// tip | "Dica"
```

File: docs/pt/docs/how-to/graphql.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+E também na documentação sobre <a href="https://strawberry.rocks/docs/integrations/fastapi" class="external-link" target="_blank">Strawberry com FastAPI</a>.
+
+## Antigo `GraphQLApp` do Starlette
+
+Versões anteriores do Starlette incluiam uma classe `GraphQLApp` para integrar com <a href="https://graphene-python.org/" class="external-link" target="_blank">Graphene</a>.
+
+Ela foi descontinuada do Starlette, mas se você tem código que a utilizava, você pode facilmente **migrar** para <a href="https://github.com/ciscorn/starlette-graphene3" class="external-link" target="_blank">starlette-graphene3</a>, que cobre o mesmo caso de uso e tem uma **interface quase idêntica**.
+
+/// tip
```
Comment (JoaoGustavoRogel) [inline docs/pt/docs/how-to/graphql.md:52]: You can add translation for this:
```
/// tip | "Dica"
```

**Review by JoaoGustavoRogel — CHANGES_REQUESTED** [review]

**Review by nazarepiedady — APPROVED** [review]
Despite the parts that need a little revision, the rest is good enough to be merged after correction is applied.

**Review by JoaoGustavoRogel — APPROVED** [review]
Great job!

File: docs/pt/docs/how-to/graphql.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+# GraphQL
+
+Como **FastAPI** é baseado no padrão **ASGI**, é muito fácil integrar qualquer biblioteca **GraphQL** também compatível com ASGI.
+
+Você pode combinar *operações de caminho* normais do FastAPI com GraphQL na mesma aplicação.
```
Comment (ceb10n) [inline docs/pt/docs/how-to/graphql.md:5]: although "operações de caminho" is also used, the most common translation in the docs is "operações de rota". I think it's nice to keep de standard 😃

File: docs/pt/docs/how-to/graphql.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+# GraphQL
+
+Como **FastAPI** é baseado no padrão **ASGI**, é muito fácil integrar qualquer biblioteca **GraphQL** também compatível com ASGI.
```
Comment (ceb10n) [inline docs/pt/docs/how-to/graphql.md:3]: I would use *o* before FastAPI:

"Como **o** FastAPI é baseado..."

and:

...GraphQL que também **é** compatível com **o** ASGI

File: docs/pt/docs/how-to/graphql.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+* <a href="https://tartiflette.io/" class="external-link" target="_blank">Tartiflette</a>
+    * Com <a href="https://tartiflette.github.io/tartiflette-asgi/" class="external-link" target="_blank">Tartiflette ASGI</a> para fornecer integração ASGI
+* <a href="https://graphene-python.org/" class="external-link" target="_blank">Graphene</a>
+    * Com <a href="https://github.com/ciscorn/starlette-graphene3" class="external-link" target="_blank">starlette-graphene3</a>
+
+## GraphQL com Strawberry
+
+Se você precisar ou quiser trabalhar com **GraphQL**, <a href="https://strawberry.rocks/" class="external-link" target="_blank">**Strawberry**</a> é a biblioteca **recomendada** pois tem o design mais próximo ao design do **FastAPI**, ela é toda baseada em **type annotations**.
+
+Dependendo no seu caso de uso, você pode preferir usar uma biblioteca diferente, mas se você me perguntasse, eu provavelmente sugeriria que você experimentasse o **Strawberry**.
```
Comment (ceb10n) [inline docs/pt/docs/how-to/graphql.md:34]: I would change **no** for **do**:

"Dependendo **do** seu caso de uso..."

**Review by ceb10n — APPROVED** [review]
Thank You Ananda 💯 

I've made just some few considerations. None of them are blocking, and you can change if you feel it's right.

Thank you 🚀

File: docs/pt/docs/how-to/graphql.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+Dependendo do seu caso de uso, você pode preferir usar uma biblioteca diferente, mas se você me perguntasse, eu provavelmente sugeriria que você experimentasse o **Strawberry**.
+
+Aqui está uma pequena prévia de como você poderia integrar Strawberry com FastAPI:
+
+```Python hl_lines="3  22  25-26"
+{!../../../docs_src/graphql/tutorial001.py!}
+```
+
+Você pode aprender mais sobre Strawberry em <a href="https://strawberry.rocks/" class="external-link" target="_blank">Strawberry documentation</a>.
+
```
Comment (marcelomarkus) [inline docs/pt/docs/how-to/graphql.md:43]: Suggestion. ✨
`em Strawberry documentation` to `na documentação do Strawberry`

**Review by ceb10n — APPROVED** [review]
❤️

**Comment by alejsdev** [comment]
Thanks for your contribution! @AnandaCampelo :nerd_face: :rocket:

**Pull request merged**